dwh/query_lib_biq_query_examples.ipynb

```diff
@@ -39,146 +40,20 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 3,
+   "execution_count": null,
```

Comment:
This is because this is a `markdown` block.

BTW, do we want to store ipynb script with output or w/o output. Currently, they are stored without output.

Reply:
We store them _with_ output. This makes it easy to browse the notebook (without running it, even in GitHub interface). BTW, have you removed `query_lib_biq_query_examples.ipynb` in the latest commit?